admin/CF7_AntiSpam_Admin_Customizations.php

```diff
@@ -1462,6 +1474,55 @@ public function cf7a_honeyform_position_callback() {
 		);
 	}
 
+	public function cf7a_honeyform_excluded_pages_callback() {
+		$args = array(
+			'post_type' => 'page', // change this to the post type you're querying
+			'fields' => 'ids', // get all posts
+		);
+		$query = new WP_Query($args);
+
+		$options = '';
+
+		if ($query->have_posts()) {
+			while ($query->have_posts()) {
+				$query->the_post();
+				$options .= '<option value="' . get_the_ID() . '">' . get_the_title() . '</option>';
+			}
+		}
+
+		$admin_options = get_option( 'cf7a_options' );
+		$excluded = $admin_options['honeyform_excluded_pages'];
+		$str_excluded = '';
+		if ( is_array($excluded) ) {
+			foreach ($excluded as $entry) {
+				$str_excluded .= '<option selected="true" value="' . $entry . '">' . get_the_title($entry) . '</option>';
+			}
+		}
+		wp_reset_postdata();
+		printf(
+			'<div class="honeyform-container">
+						 <div class="row">
+							  <div class="add">
+								<select name="add" multiple class="form-control add-select">
+								  %s
+								</select>
+								<br/>
+								<div class="add-list">Add ></div>
+							  </div>
+							  <div class="remove">
+								<select id="honeyform_excluded_pages" name="cf7a_options[honeyform_excluded_pages][]" multiple="multiple" class="form-control remove-select" >
+								%s
+								</select>
+								<br/>
+								<div class="remove-list">< Remove</div>
+							  </div>
+						 </div>
+					 </div>',
+					$options,
+					$str_excluded
+		);
```

Comment:
```suggestion
		printf(
			'<div class="honeyform-container">
						 <div class="row">
							  <div class="add">
								<select name="add" multiple class="form-control add-select">
								  %s
								</select>
								<div class="honeyform-action add-list">%s ></div>
							  </div>
							  <div class="remove">
								<select id="honeyform_excluded_pages" name="cf7a_options[honeyform_excluded_pages][]" multiple="multiple" class="form-control remove-select" >
								%s
								</select>
								<div class="honeyform-action remove-list">< %s</div>
							  </div>
						 </div>
					 </div>',
					$options,
					__( 'Add', 'cf7-antispam' ),
					$str_excluded,
					__( 'Remove', 'cf7-antispam' ),
		);
```

please provide the translated text for each one added

Reply:
FYI, you can even add an emoji's arrow (like this one ➡️) or the unicode arrow (➔)